doc-botonic/docs/components/components-whatsapptemp.md

```diff
@@ -0,0 +1,45 @@
+---
+id: whatsapptemp
```

Comment:
consistent ids: can this be renamed to just whatsapp-template? remember to change all references to `whatsapptemp` if so